bait-files/src/main/webapp/review.js

```diff
@@ -0,0 +1,14 @@
+function getPanels() {
+  const acc = document.getElementsByClassName('accordion');
+  for (let i = 0; i < acc.length; i++) {
```

Comment:
Prefer a change to "for(let x of/in accordion)", where "x" is named appropriately, if that's possible?